cds_dojson/marc21/models/books/book.py

```diff
@@ -126,24 +129,6 @@ class CDSBook(CDSOverdoBookBase):
         '963__a',
         '964__a',
         '981__a',
-
-        # '65017a',
```

Comment:
why were are all these commented fields here?

Reply:
I forgot to remove them when I was testing if the JSONs are generating from real data, now we don't need them as we get feedback about which ones should be ignored